content/projects/oop/animals2/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Animals 2!
+ready: true
+---
+
+This Challenge should test your Unit Testing knowledge using JUnit
+
+1. JUnit
+
+In this challenge you will update your current Animals project and add unit tests to the project
+
+1. Create a class called `AnimalTests`
+2. Update `Animal` super class `eats()` function to return a String, "Food".
+3. Update `Dog` class `sounds()` function to return a a `String`, "Bark".
+4. Update `Cat` class `sounds()` function to return a a `String`, "Meow".
+5. in `AnimalTests` create a `DogSoundTest()` Method
+
+```
+// Java
+
+Dog dog = new Dog()
```

Comment:
This instantiation should either be 
```
Animal dog = new Dog(); or
? dog = new Dog();
```
Because the function eat() is only available in the super class eat therefore the instantiation should be cast to it same with Cat in line 26